dist/index.js

```diff
@@ -1282,7 +1287,10 @@ function getSource(settings) {
                 core.endGroup();
             }
             // Sparse checkout
-            if (settings.sparseCheckout) {
+            if (!settings.sparseCheckout) {
+                yield git.disableSparseCheckout();
+            }
+            else {
                 core.startGroup('Setting up sparse checkout');
```

Comment:
do you want to pop the `.startGroup` / `.endGroup` calls outside of the if?

Consider `core.startGroup('Applying sparse checkout options');`

Reply:
No, because we're not actually setting up a sparse checkout. We're _disabling_ it. But saying that would only make sense if there was a worktree already, otherwise there'd be no sparse checkout to disable.